content/getting-started/setup/upgrades/index.md

```diff
@@ -8,4 +8,63 @@ dnnversion: 09.04.03
+
+## Step 1 - Prepare
+Before starting the upgrade process, it is recommended to have a very clear understanding of all upgrade requirements and expectations. Here are some tips to help prepare.
+
+* Know the current version of DNN Platform.
+
+* Ensure **AutoUpgrade** setting is "False". This setting can be found within **AppSettings** section of the `web.config` file in the root. Setting this to "False" will prevent the upgrade process from triggering by normal site visitors. 
+
+* Know the desired version of DNN Platform and download the appropriate "Upgrade" ZIP package from [the official repository on GitHub](https://github.com/dnnsoftware/DNN.Platform/releases) (which can be found in the **Assets** section for the desired version release).
+
+* Know the [suggested upgrade path](xref:setup-upgrades-suggested-upgrade-path) from the current version of DNN Platform and download all appropriate "Upgrade" ZIP packages from the above referenced GitHub link.
+
+* Know any changes to the minimum requirements for the desired version of DNN Platform and plan for implementation accordingly. 
+
+* Know upgrade requirements (if applicable) for any installed third-party extensions and plan for implementation accordingly.
+
+* If possible, plan to test the upgrade in a non-production environment prior to attempting in production.
+
+## Step 2 - Take Website Offline
+When ready to begin the upgrade process, take the website offline to avoid any access or changes to the site by other users. This can be done by adding an `app_offline.htm` file to the root of the site. This is an ASP.NET feature and the HTML file can contain any HTML you see fit to communicate with site visitors about the site unavailability. 
+
+> [!NOTE]
+> This also prevents access to any other site pages. Any attempts will be redirected to the base URL and will display the HTML message contained in the `app_offline.htm` file.
+
+## Step 3 - Backup Files & Database
+It is recommended to perform this step in a "native" fashion.
+
+All files, folders and subfolders (root and below) can be manually copied to a temporary location outside of the website directory structure.
+
+The database can be backed up using Microsoft SQL Server Management Studio (SSMS).
+
+If "native" access to the server is not possible, there other options depending on the hosting environment and available third-party extensions.  Following is a short list of alternate backup options.
+
+* Secure File Transfer Protocol (SFTP) or File Transfer Protocol (FTP) in combination with remote Microsoft SQL Server Management Studio (SSMS).
+* Control Panel (e.g., Plesk)
+* DNN Backup (3rd-party module by Evotiva)
+
+## Step 4 - Apply Upgrade Package
+Extract the desired "Upgrade" ZIP package to the root of the website. (_If the "Upgrade" ZIP file was already extracted to a temporary location, then simply copy the extracted files to the root of the website._) If prompted, chose the option to overwrite/merge any files/folders.
```

Comment:
Add an `Important` note that the zip file(s) need to be unblocked before unzipping (along with instructions on how to do that).

Reply:
These are "basic" instructions, so wasn't really planning to go into that level of detail.  If you think it is necessary, feel free to add.  I have honestly never had to "unblock" these files, so I personally consider that more of an edge case.  I could be wrong though.

Reply:
I am adding this in a new PR, but will try to keep it as concise as possible.